bridge/actions/About.cpp

```diff
@@ -0,0 +1,36 @@
+#include "ApiActions.h"
```

Comment:
actions folder name is a bit generic as well. 
These are mainly about http endpoint implementations, right?

Reply:
Renamed to endpointActions.